tests/integration/ceph_upgrade_test.go

```diff
@@ -452,6 +452,9 @@ func (s *UpgradeSuite) upgradeToMaster() {
 		return
 	}
 
+	require.NoError(s.T(), s.installer.InstallCSIOperator())
+	require.NoError(s.T(), s.installer.SetOperatorSetting("ROOK_USE_CSI_OPERATOR", "false"))
```

Comment:
```suggestion
	require.NoError(s.T(), s.installer.SetOperatorSetting("ROOK_USE_CSI_OPERATOR", "false"))
```

I cherry-picked the whole commit, but made this `false` since we don't want to enable yet

Reply:
Good idea, sounds perfect to pull that commit in. Could you just change the commit message so it removes all the messages about cherry-picking and add me as `Co-authored-by`? Since this is the first time that commit will be merged, it's good to have a clean commit message, instead of looking like it was backported. Thanks!